.buildkite/core.rayci.yml

```diff
@@ -47,6 +47,7 @@ steps:
       - oss
     instance_type: medium
     commands:
+      - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
```

Comment:
maybe write a bash script or something for this?

or support running test scripts in `ray_ci` wrapper outside a container.

Reply:
as discussed offline, since the job only runs one test, if you fixed the other two (on the flaky dashboard), this can also be reflected as a buildkite test job failure.

Reply:
If I am seeing this line again, I am going to ask for a wrapper.